stac_fastapi/sqlalchemy/stac_fastapi/sqlalchemy/transactions.py

```diff
@@ -35,11 +35,19 @@ class TransactionsClient(BaseTransactionsClient):
     )
 
     def create_item(
-        self, item: stac_types.Item, **kwargs
+        self, model: Union[stac_types.Item, stac_types.ItemCollection], **kwargs
```

Comment:
I've updated this to reflect the fact that the model is *either* `Item` or `ItemCollection`